src/transformers/models/beit/modeling_beit.py

```diff
@@ -361,6 +361,52 @@ def forward(
         return outputs
 
 
+class BeitSdpaSelfAttention(BeitSelfAttention):
+    def forward(
+        self,
+        hidden_states: torch.Tensor,
+        head_mask: Optional[torch.Tensor] = None,
+        output_attentions: bool = False,
```

Comment:
It would be great to add a fallback to `eager` attention if `output_attentions=True`, see other models for details, e.g. CLIP/SigLIP.

Reply:
Oh yes thank you for reminding 😄

Reply:
@OmarManzoor we should also call `super().forward(...)` together with warning message 🤗

Reply:
Sorry for missing

Reply:
no worries